ansible_collections/arista/avd/plugins/modules/batch_template.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 Arista Networks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+DOCUMENTATION = r"""
+---
+module: batch_template
+version_added: "4.0.0"
+author: Arista Ansible Team (@aristanetworks)
+short_description: Render Jinja2 template on multiple items and write result to individual files.
+description:
```

Comment:
Should mention somewhere that the file are created with hardcoded permissions `0o664`